src/game/rumble.c

```diff
@@ -20,10 +20,10 @@ OSMesgQueue gRumbleThreadVIMesgQueue;
 struct RumbleData gRumbleDataQueue[RUMBLE_QUEUE_SIZE];
 struct RumbleSettings gCurrRumbleSettings;
 
-s32 sRumblePakThreadActive = FALSE; // Set to TRUE when the rumble thread starts.
-s32 sRumblePakActive = FALSE;
-s32 sRumblePakErrorCount = 0;
-s32 gRumblePakTimer = 0;
+_Bool sRumblePakThreadActive = FALSE; // Set to TRUE when the rumble thread starts.
```

Comment:
`_Bool` doesn't exist anywhere else in the repo. Not sure if it comes with any downsides, but I might just stick with u8 for consistency sake. (At minimum, make sure it compiles with GCC 9, or it's definitely a no-go.)